client/jetpack-connect/controller.js

```diff
@@ -2,31 +2,33 @@
 /**
  * External Dependencies
  */
-import ReactDom from 'react-dom';
 import React from 'react';
-import page from 'page';
+import ReactDom from 'react-dom';
 import Debug from 'debug';
-import { translate } from 'i18n-calypso';
+import page from 'page';
 import { get, isEmpty } from 'lodash';
+import { translate } from 'i18n-calypso';
 
 /**
  * Internal Dependencies
  */
+import analytics from 'lib/analytics';
+import CheckoutData from 'components/data/checkout';
+import i18nUtils from 'lib/i18n-utils';
 import JetpackConnect from './main';
-import JetpackNewSite from './jetpack-new-site/index';
 import JetpackConnectAuthorizeForm from './authorize-form';
-import { setSection } from 'state/ui/actions';
-import { renderWithReduxStore } from 'lib/react-helpers';
-import { JETPACK_CONNECT_QUERY_SET } from 'state/action-types';
-import userFactory from 'lib/user';
+import JetpackNewSite from './jetpack-new-site/index';
 import jetpackSSOForm from './sso';
-import i18nUtils from 'lib/i18n-utils';
-import analytics from 'lib/analytics';
+import Plans from './plans';
+import PlansLanding from './plans-landing';
 import route from 'lib/route';
-import { setDocumentHeadTitle as setTitle } from 'state/document-head/actions';
+import userFactory from 'lib/user';
 import { getSelectedSiteId } from 'state/ui/selectors';
 import { isJetpackSite } from 'state/sites/selectors';
-import PlansLanding from './plans-landing';
+import { JETPACK_CONNECT_QUERY_SET } from 'state/action-types';
+import { renderWithReduxStore } from 'lib/react-helpers';
+import { setDocumentHeadTitle as setTitle } from 'state/document-head/actions';
+import { setSection } from 'state/ui/actions';
```

Comment:
`CheckoutData` and `Plans` have been hoisted and converted to `import`s. The rest have just been alphabetized.